### DISCUSSION TITLE
Freeplane Privacy Policy

### DISCUSSION THREAD

**Discussion by nerradnot** [freeplane/freeplane#2111] [Help]
Hi,

Is there a privacy policy for Freeplane. If so, can someone point me in the right direction.

Thank you.

**Comment by dpolivaev** [comment]
I do not understand your question

**Comment by nerradnot** [comment]
Is there any privacy policy for this software, i.e. is any data collected
by using the software, if so where is it stored, how is it used, who is it
shared with etc.
